CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing to Hermes 
+
+To ensure consistency and compatibility within our tools, pipeline developers are encouraged to adhere to the following
+guidelines when contributing to Hermes.
+
+## Deliverables Format
+
+Workflows must clearly define their deliverables in a [standardised format](https://atlas.scilifelab.se/infrastructure/dataflow/workflow/files_delivery/).
+
+## Tags Format
+
+Tags provide additional context or metadata about the workflow output files and play a crucial role in organising and 
+categorising them within the repository and [CG].
+
+When adding or modifying tags, please follow these guidelines:
+  - Use descriptive tag names that accurately represent the file's purpose, domain, or functionality.
+  - Tags should follow a consistent syntax, such as using lowercase letters and hyphens to separate words (e.g., 
+    `qc-metrics`, `vcf-snv-clinical`).
+  - Tags should uniquely describe a file to enable individual retrieval by other tools if needed. Therefore, strictly 
+    avoid adding the same tag set to multiple files, as this can lead to ambiguity.
+  - Ensure that the deliverables tag specification is compatible with the format required by CG for long-term storage
+    (`long-term-storage`) and delivery (`clinical-delivery`).
```

Comment:
can we use the `deliver` tag as we were doing instead of `clinical-deliver` to be able to deliver previous analyses? why creating a new one?

Reply:
it's not uncommon to be asked to deliver old cases but since it wasn't possible before the cases had to be re-run again (if I'm not mistaken but @karlnyr can correct me if I'm wrong)
I see your point about unexpected behaviours, so agree with your approach then!

Reply:
We'll continue to have issues with the delivery of old cases that lack cram files for example, which are eventually removed (along with other analysis files we may not store long-term). I suppose it's a limitation of re-delivering old analyses... we can't store everything